common/source/docs/common-auxiliary-functions.rst

```diff
@@ -0,0 +1,633 @@
+|        68            | STABILIZE mode             |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        69            | POSHOLD mode               |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        70            | ALTHOLD mode               |    X     |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        71            | FLOWHOLD mode              |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        72            | CIRCLE mode                |    X     |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        73            | DRIFT mode                 |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        74            | Sailboat Motor 3Pos Sw     |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        75            | Surface Tracking Up/Down   |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        76            | STANDBY mode               |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        77            | TAKEOFF mode               |          |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        207           | Mainsail (Sailboat)        |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+
+
+Description of Features
+=======================
+
+.. note:: in descriptions below, LOW and HIGH for a channel refers to PWM <1200us and >1800us, respectively.
+
+Mode Switches
+-------------
+
+Any feature ending with "mode" provides the ability to switch the vehicle into that mode by setting the RC channel to high. This is an "edge-triggered" event, ie the change from low to high is the cause of the mode change, not the level. To re-trigger the mode change requires changing the channel's PWM from low to high.
+
+.. note:: If mapped to a three-position switch then the SUPERSIMPLE mode function will allow **SUPERSIMPLE** and **SIMPLE** modes to be enabled using the high and middle switch positions, respectively (a two-position switch will enable/disable SUPERSIMPLE mode only). :ref:`See here for more details<simpleandsuper-simple-modes>` .
+
+
+
+Other functions are:
+
+.. raw:: html
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Option</th>
+   <th>Description</th>
+   </tr>
+   <tr>
+   <td><strong>Flip</strong></td>
```

Comment:
I'm a big fan of avoiding duplication in the wiki so I think we can skip this description for Flip because we already have it on the Flip flight mode page: http://ardupilot.org/copter/docs/flip-mode.html#flip-mode

Reply:
agree re duplication, but I think that it really should be an action and not a mode, and I think it should also be removed from the FLTMODEx Params....and this could refer to that page (with a tweak to the page)

Reply:
since its a real mode, I will change to refer to the existing mode wiki page...